samples/subsys/zbus/msg_subscriber/README.rst

```diff
@@ -0,0 +1,78 @@
+.. zephyr:code-sample:: zbus-msg-subscriber
+   :name: zbus Message subscriber
```

Comment:
```suggestion
   :name: Message subscriber
```

FWIW I added the "zbus" prefix only to "zbus Hello World" but I don't think other samples require it as they already live under the "zbus" samples anyways. But now that I think about it I guess maybe "_zbus_ Hello World" wasn't needed either? 🤷‍♂️

Reply:
It is interesting because that must be different from plain hello world in text search.